rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/importexport/Sw360ImportExportService.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright Siemens AG, 2024-2025.
+ * Part of the SW360 Portal Project.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.sw360.rest.resourceserver.importexport;
+
+import static org.eclipse.sw360.importer.ComponentImportUtils.getFlattenedView;
+import static org.eclipse.sw360.importer.ComponentImportUtils.getReleasesById;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.thrift.TException;
+import org.eclipse.sw360.datahandler.common.SW360Utils;
+import org.eclipse.sw360.datahandler.permissions.PermissionUtils;
+import org.eclipse.sw360.datahandler.thrift.ReleaseRelationship;
+import org.eclipse.sw360.datahandler.thrift.SW360Exception;
+import org.eclipse.sw360.datahandler.thrift.ThriftClients;
+import org.eclipse.sw360.datahandler.thrift.ThriftUtils;
+import org.eclipse.sw360.datahandler.thrift.attachments.Attachment;
+import org.eclipse.sw360.datahandler.thrift.components.Component;
+import org.eclipse.sw360.datahandler.thrift.components.ComponentService;
+import org.eclipse.sw360.datahandler.thrift.components.ComponentService.Iface;
+import org.eclipse.sw360.datahandler.thrift.components.Release;
+import org.eclipse.sw360.datahandler.thrift.users.User;
+import org.eclipse.sw360.datahandler.thrift.users.UserGroup;
+import org.eclipse.sw360.exporter.CSVExport;
+import org.eclipse.sw360.importer.ComponentAttachmentCSVRecord;
+import org.eclipse.sw360.importer.ComponentAttachmentCSVRecordBuilder;
+import org.eclipse.sw360.importer.ComponentCSVRecord;
+import org.eclipse.sw360.importer.ReleaseLinkCSVRecord;
+import org.eclipse.sw360.importer.ReleaseLinkCSVRecordBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.stereotype.Service;
+import org.springframework.util.FileCopyUtils;
+
+import com.google.common.collect.ImmutableList;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class Sw360ImportExportService {
+    private final Logger log = LoggerFactory.getLogger(this.getClass());
+
+    public void getDownloadCsvComponentTemplate(User sw360User, HttpServletResponse response) throws TException, IOException {
+        String fileConstant="ComponentsReleasesVendorsSample.csv";
```

Comment:
"**fileConstant**" : Since its used only once, no need to create a separate String parameter.